### PR TITLE
chore: clarify logic for request body processors.

### PR DESCRIPTION
**Pull request by jcchavezs**
Ping @M4tteoP 

> Thank you for contributing to Coraza WAF, your effort is greatly appreciated
> Before submitting check if what you want to add to `coraza` list meets [quality standards](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#quality-standards) before sending pull request. Thanks!

**Make sure that you've checked the boxes below before you submit PR:**

- [ ] My code includes positive and negative tests.
- [ ] I have an appropriate description with correct grammar.
- [ ] I have read [Contribution guidelines](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/github.com/corazawaf/coraza/v3sso/coraza-waf/blob/master/CONTRIBUTING.md#quality-standards).
- [ ] My code is properly linted and passes pre-commit tests.

Thanks for your contribution :heart:

### CODE REVIEW

File: transaction.go [hunk 1]
```diff
@@ -561,27 +561,28 @@ func (tx *Transaction) ProcessRequestBody() (*types.Interruption, error) {
...
+			tx.WAF.Rules.Eval(types.PhaseRequestBody, tx)
+			return tx.Interruption, nil
+		}
+
+		// We force URLENCODED if we have an empty RBP and ForceRequestBodyVariable
 		rbp = "URLENCODED"
 		tx.Variables.ReqbodyProcessor.Set(rbp)
 	}
 	tx.WAF.Logger.Debug("[%s] Attempting to process request body using %q", tx.ID, rbp)
-	rbp = strings.ToLower(rbp)
```
Comment (jptosso) [inline transaction.go:573]: rbp must be handled as lowercase

Comment (jcchavezs) [reply transaction.go:573]: https://github.com/corazawaf/coraza/pull/441/files#diff-63ab601287b8b9c040760fe0bdd288f55b73f37cd7e4f1e519bea2bd43a18bbaL579 already lowercase this.

**Comment by jptosso** [comment]
@jcchavezs I think it was fixed in a different PR?

**Comment by jptosso** [comment]
ping @jcchavezs

**Comment by fzipi** [comment]
ping @jcchavezs ? do we need to close this one?

**Comment by fzipi** [comment]
ping @jcchavezs ? last call.

**Pull request closed without merging**